Sources/Widgets/Representations/ResliceCursorContextRepresentation/index.js

```diff
@@ -154,19 +155,35 @@ function vtkResliceCursorContextRepresentation(publicAPI, model) {
     ctxVisible,
     hVisible
   ) => {
-    const visiblity =
+    const visibility =
       renderingType === RenderingTypes.PICKING_BUFFER
         ? wVisible
         : wVisible && hVisible;
 
     publicAPI.getActors().forEach((actor) => {
       actor.getProperty().setOpacity(model.opacity);
+      let actorVisibility = visibility;
+
+      // Conditionally display rotation handles
       if (publicAPI.getRotationActors().includes(actor)) {
-        actor.setVisibility(visiblity && model.rotationEnabled);
-      } else {
-        actor.setVisibility(visiblity);
+        actorVisibility = visibility && model.rotationEnabled;
+      }
+
+      // Conditionally display center handle but always show it for picking
+      if (!model.showCenter && actor === model.pipelines.center.actor) {
+        actorVisibility =
+          actorVisibility && renderingType === RenderingTypes.PICKING_BUFFER;
       }
+
+      actor.setVisibility(actorVisibility);
     });
+
+    // Define center visibility
+    const centerVisibility =
+      renderingType === RenderingTypes.PICKING_BUFFER
+        ? visibility
+        : visibility && model.showCenter;
```

Comment:
this is no longer needed

Reply:
my bad. Seems that I want to go faster than I should..